tox.ini

```diff
@@ -117,6 +117,15 @@ extras =
 deps =
     astropy
     numpy
-    parfive[ftp]
+    packaging
 commands =
     python -c "import sunpy; sunpy.self_test()"
+    # Test that we can import subpackages which don't have specific extras
+    # TODO: Actually run the tests for these modules
```

Comment:
In theory (hahaa) we should be able to run the tests for these subpackages with the test and core deps installed.

Reply:
How does a sunpy test function to do these sound instead of adding lines to tox?